src/components/Checkster/feature/docs/AboutBrowserChecks.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Stack, Text } from '@grafana/ui';
+
+export function AboutBrowserChecks() {
+  return (
+    <Stack direction="column" gap={2}>
+      <Text element="p">
```

Comment:
Feels like we're missing a heading before the "about text", aren't we?

<img width="506" height="610" alt="Image" src="https://github.com/user-attachments/assets/6e87e672-dbbe-461f-9297-49b5f1b9b55d" />